client/my-sites/customer-home/cards/features/help-search/search-card.jsx

```diff
@@ -103,9 +87,23 @@ const mapStateToProps = ( state, ownProps ) => ( {
 	selectedResultIndex: getSelectedResultIndex( state ),
 	selectedResult: getInlineHelpCurrentlySelectedResult( state ),
 } );
+
+const requestInlineSearchResultsAndTrack = ( searchQuery ) =>
```

Comment:
Combining everything here might be difficult to test in the future. Would it be better to have separate concerns here for "requesting results" and "tracking the query". 

Indeed the actual result of calling `requestInlineSearchResultsAndTrack` doesn't always result in events being tracked.

Reply:
> Indeed the actual result of calling requestInlineSearchResultsAndTrack doesn't always result in events being tracked.

It doesn't track only when the query is not defined or it is an empty string, which is intentional. Is it correct to you?